libs/ddhub-client-gateway-frontend/ui/api-hooks/src/lib/messaging/uploadMessage.ts

```diff
@@ -5,6 +5,14 @@ import {
   SendMessageDto,
 } from '@dsb-client-gateway/dsb-client-gateway-api-client';
 import { useCustomAlert } from '@ddhub-client-gateway-frontend/ui/core';
+import { Subject } from 'rxjs';
+
+const subject = new Subject();
+
+export const messageDataService = {
```

Comment:
Why rxjs if you can do similar stuff with context API? Because if i understood correctly, you have some data that should be displayed in different view. Am i right?

Reply:
I have tried that before but the updated value did not appear as expected.